compiler/ast.jou

```diff
@@ -316,6 +319,8 @@ class AstExpression:
                 printf("and\n")
             case AstExpressionKind.Or:
                 printf("or\n")
+            case AstExpressionKind.Constant:
+                printf("constant\n")
```

Comment:
Should be moved up just after `GetEnumMember`, should print the value of the constant